parlai/utils/torch.py

```diff
@@ -235,3 +268,15 @@ def fp16_available() -> bool:
             'install APEX from https://github.com/NVIDIA/apex.'
         )
         return False
+
+
+class DummyEncoder(torch.nn.Module):
```

Comment:
If we're really going to put this into a util folder, I'd prefer it be called IdentityLayer.